tests/acceptance/features/apiSettings/notificationSetting.feature

```diff
@@ -444,3 +444,237 @@ Feature: Notification Settings
+                "type": "object",
+                "required": [
+                  "id",
+                  "bundleId",
+                  "settingId",
+                  "accountUuid",
+                  "resource",
+                  "collectionValue"
+                ],
+                "properties":{
+                  "id":{
+                    "pattern":"%user_id_pattern%"
+                  },
+                  "bundleId":{
+                    "pattern":"%user_id_pattern%"
+                  },
+                  "settingId":{
+                    "pattern":"%user_id_pattern%"
+                  },
+                  "accountUuid":{
+                    "pattern":"%user_id_pattern%"
+                  },
+                  "resource":{
+                    "type": "object",
+                    "required":["type"],
+                    "properties": {
+                      "type":{
+                        "const": "TYPE_USER"
+                      }
+                    }
+                  },
+                  "collectionValue":{
+                    "type": "object",
+                    "required":["values"],
+                    "properties": {
+                      "values":{
+                        "type": "array",
+                        "maxItems": 1,
+                        "minItems": 1,
+                        "uniqueItems": true,
+                        "items": {
+                          "oneOf": [
+                            {
+                              "type": "object",
+                              "required": [
+                                "key",
+                                "boolValue"
+                              ],
+                              "properties": {
+                                "key":{
+                                  "const": "in-app"
+                                },
+                                "boolValue":{
+                                  "const": false
+                                }
+                              }
+                            }
+                          ]
+                        }
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+      """
+    And user "Brian" has uploaded file "filesForUpload/filesWithVirus/<file-name>" to "<new-file-name>"
+    When user "Brian" lists all notifications
+    Then the HTTP status code should be "200"
+    And user "Brian" should not have any notification
+    Examples:
+      | dav-path-version | file-name     | new-file-name  |
+      | old              | eicar.com     | virusFile1.txt |
+      | old              | eicar_com.zip | virusFile2.zip |
+      | new              | eicar.com     | virusFile1.txt |
+      | new              | eicar_com.zip | virusFile2.zip |
+      | spaces           | eicar.com     | virusFile1.txt |
+      | spaces           | eicar_com.zip | virusFile2.zip |
```

Comment:
should we check both .com and .zip files for notification? IMO, it doesn't but it needs to be a virus file